framework/src/components/storage/adapters/pgp_adapter.js

```diff
@@ -166,6 +171,10 @@ class PgpAdapter extends BaseAdapter {
 	loadSQLFile(filePath, sqlDirectory = this.sqlDirectory) {
 		const fullPath = path.join(sqlDirectory, filePath); // Generating full path;
 
+		if (_private.queryFiles[fullPath]) {
```

Comment:
PR seems good. Just mention in comments in the code with reference to `pg-promise` docs that why we needed this.